packages/zowe-explorer-ftp-extension/src/ZoweExplorerFtpApi.ts

```diff
@@ -183,10 +204,14 @@ export class FtpUssApi implements ZoweExplorerApi.IUss {
         const connection = await this.ftpClient(this.checkedProfile());
         if (connection && connection.client) {
             if (type === "directory") {
```

Comment:
I suppose it's from Zowe Explorer. If type could be `File` or `file`, how about `directory`?

Reply:
There are two places to create file in vscode, one the type is 'file', one is 'File'. During my tests I found this. No findings on directory. Thank you!